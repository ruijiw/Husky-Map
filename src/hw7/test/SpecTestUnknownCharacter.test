# Find a path using the LOLDictionary dataSet.
# This test uses the parser, the MarvelPaths2's setUpGraph
# and Dijkstra algorithm and applies to a targeted dataSet.
# This test is for the unknown characters case.

LoadGraph dictionary LOLDictionary.tsv

FindPath dictionary TEEMO UNKNOWN

FindPath dictionary UNKNOWN JINX

FindPath dictionary UNKNOWN_A UNKNOWN_B