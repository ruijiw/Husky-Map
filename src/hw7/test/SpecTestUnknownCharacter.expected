# Find a path using the LOLDictionary dataSet.
# This test uses the parser, the MarvelPaths2's setUpGraph
# and Dijkstra algorithm and applies to a targeted dataSet.
# This test is for the unknown characters case.

loaded graph dictionary

unknown character UNKNOWN

unknown character UNKNOWN

unknown character UNKNOWN A
unknown character UNKNOWN B